explorer/ast/value_category.h

```diff
@@ -11,9 +11,9 @@ namespace Carbon {
 // to a variable or a value. A variable can be mutated, and can have its
 // address taken, whereas a value cannot.
 enum class ValueCategory {
-  // A variable. This roughly corresponds to a C/C++ lvalue.
+  // A variable. This roughly corresponds to a C and C++ lvalue.
```

Comment:
```suggestion
  // A variable. This roughly corresponds to a C++ lvalue.
```
Value categories are different enough between C and C++ that I don't think it's helpful to lump them together here. (Two suggestions rather than one due to github limitations.)